plugins/src/webrtcsink/imp.rs

```diff
@@ -1642,24 +1737,50 @@ impl WebRTCSink {
             }
         });
 
-        let mut consumer = Consumer {
-            pipeline: pipeline.clone(),
-            webrtcbin: webrtcbin.clone(),
-            webrtc_pads: HashMap::new(),
-            peer_id: peer_id.to_string(),
-            congestion_controller: match settings.cc_heuristic {
+        let mut consumer = Consumer::new(
+            pipeline.clone(),
+            webrtcbin.clone(),
+            peer_id.to_string(),
+            match settings.cc_heuristic {
                 WebRTCSinkCongestionControl::Disabled => None,
                 WebRTCSinkCongestionControl::Homegrown => Some(CongestionController::new(
                     peer_id,
                     settings.min_bitrate,
                     settings.max_bitrate,
                 )),
             },
-            encoders: Vec::new(),
-            sdp: None,
-            stats: gst::Structure::new_empty("application/x-webrtc-stats"),
-            max_bitrate: settings.max_bitrate,
-        };
+            settings.max_bitrate,
+        );
+
+        let rtpbin = webrtcbin
+            .dynamic_cast_ref::<gst::ChildProxy>()
+            .unwrap()
+            .child_by_name("rtpbin")
+            .unwrap();
+
+        if consumer.congestion_controller.is_some() {
+            let peer_id_str = peer_id.to_string();
+            consumer.stats_sigid = Some(rtpbin.connect_closure("on-new-ssrc", true,
+            glib::closure!(@weak-allow-none element, @weak-allow-none webrtcbin
+                            => move |rtpbin: gst::Object, session_id: u32, _src: u32| {
+                    let session = rtpbin.emit_by_name::<gst::Element>("get-session", &[&session_id]).property::<glib::Object>("internal-session");
+
+                    let element = element.expect("on-new-ssrc emited when webrtcsink has been disposed?");
+                    let webrtcbin = webrtcbin.unwrap();
+                    let mut state = element.imp().state.lock().unwrap();
+                    if let Some(mut consumer) = state.consumers.get_mut(&peer_id_str) {
+                        consumer.stats_sigid = Some(session.connect("on-feedback-rtcp",
+                            false,
+                            glib::clone!(@strong peer_id_str, @weak webrtcbin, @weak element => @default-panic, move |_| {
```

Comment:
But why not also here?

Reply:
Because I didn't care at all about the arguments in that case.